packages/cli/scripts/install.js

```diff
@@ -38,23 +58,54 @@ module.exports = function (installationDir) {
       }
     },
+    cloneTheme: {
+      title: 'Copying Vue Storefront theme',
+      task: answers => execa.shell([
+        `git clone --quiet --single-branch --branch ${answers.themeBranch} https://github.com/DivanteLtd/vsf-${answers.themeName}.git ${installationDir}/src/themes/${answers.themeName}`,
+        `cd ${installationDir}/src/themes/${answers.themeName}`,
+        `git remote rm origin`
+      ].join(' && '))
+    },
+    configureTheme: {
+      title: 'Configuring Vue Storefront theme',
+      task: answers => {
+        const vsfLocalJsonPath = `${installationDir}/config/local.json`
+        const themeLocalJsonPath = `${installationDir}/src/themes/${answers.themeName}/local.json`
+
+        if (fs.existsSync(themeLocalJsonPath)) {
+          try {
+            const vsfLocalJson = fs.existsSync(vsfLocalJsonPath) ? JSON.parse(fs.readFileSync(vsfLocalJsonPath)) : {}
+            const themeLocalJson = JSON.parse(fs.readFileSync(themeLocalJsonPath))
+
+            fs.writeFileSync(vsfLocalJsonPath, JSON.stringify(Object.assign(vsfLocalJson, themeLocalJson), null, 2))
+          } catch (e) {
+            console.error('Problem with parsing or merging local.json configurations', e)
+          }
+        }
+      },
+      skip: answers => {
+        if (!fs.existsSync(`${installationDir}/src/themes/${answers.themeName}/local.json`)) {
+          return 'Missing local.json in theme'
+        }
+      }
+    },
     runInstaller: {
       title: 'Running installer',
-      task: () => spawn.execFileSync('yarn', ['installer'], {stdio: 'inherit', cwd: installationDir})
+      task: () => spawn.execFileSync('yarn', ['installer'], { stdio: 'inherit', cwd: installationDir })
     },
     getStorefrontVersions: {
-      title: 'Check avalilable versions',
+      title: 'Check available versions',
       task: () => execa.stdout('git', ['ls-remote', '--tags', 'https://github.com/DivanteLtd/vue-storefront.git']).then(result => {
-          allTags = result.match(/refs\/tags\/v1.([0-9.]+)(-rc.[0-9])?/gm).map(tag => tag.replace('refs/tags/', ''))
-          allTags = semverSort.desc(allTags)
-          execa.stdout('git', ['ls-remote', '--heads', 'https://github.com/DivanteLtd/vue-storefront.git']).then(branches => {
-            let rcBranches = branches.match(/refs\/heads\/release\/v1.([0-9.]+)/gm).map(tag => tag.replace('refs/heads/', ''))
-            availableBranches = [...rcBranches, ...availableBranches]
-          })
+        allTags = result.match(/refs\/tags\/v1.([0-9.]+)(-rc.[0-9])?/gm).map(tag => tag.replace('refs/tags/', ''))
+        allTags = semverSortDesc(allTags)
+        execa.stdout('git', ['ls-remote', '--heads', 'https://github.com/DivanteLtd/vue-storefront.git']).then(branches => {
+          let rcBranches = branches.match(/refs\/heads\/release\/v1.([0-9.x]+)/gm).map(tag => tag.replace('refs/heads/', ''))
```

Comment:
Changed regex to include `x` in `v1.([0-9.x]+)` is needed to properly match `release/v1.10.x` branch.